src/ht.c

```diff
@@ -3,9 +3,11 @@
 #include "ht.h"
 #include "sdb.h"
 
+#define LOAD_FACTOR 0.9
```

Comment:
0.9? load_factor is an ut32, not a float. and i would prefer to not depend on fpu

Reply:
oh! i see you deleted the variable... so there's no assignment. but this makes the fpu play into the game, and i would prefer to not depend on it for performance reasons